### ISSUE TITLE
3.0.5 broke package import

### ISSUE THREAD

**Issue by balbesina** [ablanco/jquery.pwstrength.bootstrap#162]
Prior 3.0.5 jquery was referenced ">= 3.0.0", now it is "^3.4.1".
Since jquery 3.5 release pwstrength started using 3.4.1, while all my other bundle is using 3.5.0.
```js
global.$ = global.jQuery = require('jquery');
console.log($.fn.jquery) // "3.5.0"
```

inside the node_modules/pwstrength-bootstrap/dist/pwstrength-bootstrap.js (function($){...})(jQuery)
```js
console.log($.fn.jquery) // "3.4.1"
```

as a result in window I don't have pwstrength initialized, bcoz it was injected into 3.4.1 instead of global 3.5.0
```js
$.fn.jquery // "3.5.0"
$.fn.pwstrength // undefined
```
When I rollback pwstrength to "=3.0.4" it initializes w/o issues.
Mb you should use ">= 3.4.1" instead of "^3.4.1" for jquery dependency?

**Comment by ablanco** [comment]
To be honest, I updated it because of an automated security alert from Github and I didn't give it much thought.

The dependencies don't make much sense, the plugin it's supposed to work with jQuery 1.7 and Bootstrap 2. Those are the actual minimum requirements.

```javascript
    "dependencies": {
        "bootstrap": ">=2.0.0",
        "jquery": ">=1.7.0"
    },
``` 

That makes much more sense to me, and I will just ignore the Github security alerts.

I've released a 3.0.7 version with this change.

**Comment by balbesina** [comment]
thanks, that fixed the bug until 3.5.1 came out. for reasons unknown your plugin stuck to 3.5.0 and ignores my 3.5.1 global jquery from package.json.
mb you need to add it to `peerDependencies` instead of `dependencies` ?
https://nodejs.org/es/blog/npm/peer-dependencies

**Comment by ablanco** [comment]
Interesting, I didn't know about `peerDependencies`. I've changed it to use them.
